transitions/multi_report_alerts.js

```diff
@@ -85,8 +85,8 @@ const generateMessages = (alert, phones, latestReport, countedReportsIds, newRep
 // Recipients format examples:
 // [
 //    '+254777888999',
-//    'countedReport.contact.parent.parent.contact.phone',   // returns string
-//    'countedReport.contact.parent.parent.alertRecipients', // returns string array
+//    'new_report.contact.parent.parent.contact.phone',   // returns string
```

Comment:
Just a comment, but this should have the item, right? eg `new_report.0.contact`...